Product/Production/Services/DocumentQueryCore/src/test/java/gov/hhs/fha/nhinc/docquery/audit/transform/DocQueryAuditTransformsTest.java

```diff
@@ -167,6 +170,12 @@ private void assertParticiopantObjectIdentification(LogEventRequestType auditReq
         assertEquals(DocQueryAuditTransformsConstants.PARTICIPANT_QUERY_OBJ_ID_TYPE_DISPLAY_NAME,
             auditRequest.getAuditMessage().getParticipantObjectIdentification().get(1).
             getParticipantObjectIDTypeCode().getDisplayName());
+        assertNull(null, auditRequest.getAuditMessage().getParticipantObjectIdentification().get(0).
```

Comment:
Don't use null as first parameter -- either make a proper assertion failure message (e.g., something like "ParticipantObjectName must be null"), or use the single-arg version of assertNull.

http://junit.org/apidocs/org/junit/Assert.html#assertNull(java.lang.String,%20java.lang.Object)
http://junit.org/apidocs/org/junit/Assert.html#assertNull(java.lang.Object)